cli/src/cli.rs

```diff
@@ -11,6 +11,43 @@ pub struct Cli {
     command: Commands,
 }
 
+#[derive(Debug, Subcommand)]
+pub enum CreateCommands {
+    /// An extra-agenda transaction that delegates the consensus voting power.
+    TxDelegate {
+        delegator: String,
+        delegatee: String,
+        /// Whether to delegate the governance voting power too.
+        governance: bool,
+        proof: String,
+    },
+    /// An extra-agenda transaction that undelegates the consensus voting power and
+    /// the governance voting power (if delegated).
+    TxUndelegate { delegator: String, proof: String },
+    /// An extra-agenda transaction that reports a misbehaving validator.
+    TxReport, // TODO
+    /// A block waiting for finalization.
+    Block,
+    /// An agenda wating for governance approval.
+    Agenda,
+}
+
+#[derive(Debug, Subcommand)]
+pub enum SignCommands {
+    TxDelegate {
+        delegatee: String,
+        /// Whether to delegate the governance voting power too.
+        governance: bool,
+        target_height: u64,
```

Comment:
```suggestion
        target_height: BlockHeight,
```

Reply:
Let's fix this in the later commit. This cause an unnecessary git conflict after my next work.

Reply:
Then maybe a TODO? If TODO causes a git conflict then skip and fix it later.

Reply:
I will fix soon. No need for formality here.